File: src/main/kotlin/tech/diggle/apps/accomodation/elections/ElectionService.java
package tech.diggle.apps.accomodation.elections;



import java.util.Date;
import java.util.List;


public interface ElectionService {
    List<Election> listAll();

    Election createElection(Election election);

    boolean deleteElection(Election election);

    Date findElectionByElectionDate(Date election);

    Election findElectionByElectionType(String election);


}

File: src/main/kotlin/tech/diggle/apps/accomodation/election/ElectionService.kt
package tech.diggle.apps.accomodation.election

interface ElectionService {
    fun getAll(): List<Election>
    fun get(id: Long): Election
    fun getResults(id: Long): Any
    fun add(election: Election): Election
    fun update(election: Election): Election
    fun find(election: Election): Election?
}